data/part-1/5-calculating.md

```diff
@@ -783,15 +783,15 @@ The last example produces an incorrectly rounded result, because the integer div
 
 <!-- Jos jakolausekkeen tulos asetetaan kokonaislukutyyppiseen muuttujaan, on tulos automaattisesti kokonaisluku. -->
 
-If the result of a division is assigned to an integer-type variable, the result is automatically an integer.
+If the floating point result of a division is assigned to an integer-type variable, the result is ***NOT automatically an integer*** -- it has to be *casted* to `int` (***demotion*** `double` -> `int` with lost of decimal part of result {*possible lossy conversion from `double` to `int`*}!!!), like this:
```

Comment:
Same as above, this tries to condense a lot of information into a small amount of space, and may confuse students. Unnecessary use of exclamation marks and some typos (lost of decimal --> loss of decimal part)

Reply:
yes, "loss" is more appropriate, because "lost" would be the adjective, not the noun in this case,
but "with lost decimal part of the result" would be OK, I think. :thinking: :smile:
[e.g. "a lost child"; "lost friends"; "his lost book"; "lost opportunities"]

P.S. the exclamation marks could be thrown away. :small_airplane: